metals/src/main/scala/scala/meta/internal/metals/ClassPath.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.metals
+import scala.meta.internal.semanticdb.SymbolOccurrence
+
+object ClassPath {
+  final case class TypeSignature(value: String) extends AnyVal
```

Comment:
Did you measure that allocations from this class were causing problems? If not, let's remove `AnyVal`
```suggestion
  final case class TypeSignature(value: String)
```